backend/cmmgen_state.ml

```diff
@@ -82,7 +98,9 @@ let rank_structured_constant = function
   | Const_float_array _ -> 5
   | Const_string _ -> 6
   | Const_vec128 _ -> 8
-  | Const_float32 _ -> 9
+  | Const_vec256 _ -> 9
```

Comment:
(I suspect we don't care, but having
a hole in the returned values is very
likely to generate worse code.)

Reply:
Thanks - indeed that was not part of the diff.